locales/es/translation.json

```diff
@@ -1,1327 +1,1328 @@
+    "Select roles": "Seleccionar roles",
+    "Select signatures": "Seleccionar firmas",
+    "Select statuses": "Seleccionar estados",
+    "Select the instance groups for this inventory to run on.": "Seleccione los grupos de instancias en los que se ejecutará este inventario.",
+    "Select the instance groups for this job template to run on.": "Seleccione los grupos de instancias en los que se ejecutará esta plantilla de trabajo.",
+    "Select the instance groups for this organization to run on.": "Seleccione los grupos de instancias en los que se ejecutará esta organización.",
+    "Select the inventory containing the hosts you want this job to manage.": "Seleccione el inventario que contenga los servidores que desea que este trabajo administre.",
+    "Select the playbook to be executed by this job.": "Seleccionar el playbook a ser ejecutado por este trabajo.",
+    "Select the project containing the playbook you want this job to execute.": "Seleccionar el proyecto que contiene el playbook que desea ejecutar este trabajo.",
+    "Select types": "Seleccionar los tipos",
+    "Select user type": "Seleccionar el tipo de usuario",
+    "Selected": "Seleccionado",
+    "Settings": "Ajustes",
+    "Show": "Mostrar",
+    "Show changes": "Mostrar cambios",
+    "Signature": "Firma",
+    "Signature Keys": "Claves de firma",
+    "Signed": "Firmado",
+    "Signed state": "Estado firmado",
+    "similar to a jump host, hop nodes will route traffic to other execution nodes. Hop nodes cannot execute automation.": "De forma similar a un host de salto, los nodos de salto enrutarán el tráfico a otros nodos de ejecución. Los nodos de salto no pueden ejecutar la automatización.",
+    "Skip tags": "Omitir etiquetas",
+    "Skip tags are useful when you have a large playbook, and you want to skip specific parts of a play or task. Use commas to separate multiple tags. Refer to the documentation for details on the usage of tags.": "Omitir etiquetas es útil cuando se posee de un playbook largo y desea omitir algunas partes de una jugada o tarea. Utilice comas para separar varias etiquetas. Consulte la documentación para más detalles sobre el uso de las etiquetas.",
+    "Smart host filter": "Filtro de host inteligente",
+    "Smart inventory": "Inventario inteligente",
+    "Social": "Social",
+    "Soft delete hostnames_one": "Borrar nombre de host",
+    "Soft delete hostnames_other": "Borrado suave de nombres de host",
+    "Something went wrong": "Algo salió mal",
+    "Source": "Fuente",
+    "Source control branch": "Rama de fuente de control",
+    "Source control credential": "Credencial de fuente de control",
+    "Source control refspec": "Refspec de fuente de control",
+    "Source control revision": "Revisión de fuente de control",
+    "Source control type": "Tipo de fuente de control",
+    "Source control update": "Actualización de fuente de control",
+    "Source control URL": "URL de fuente de control",
```

Comment:
I think this is a case where the literal translation could not be the right choice. 

Source control refers to SCM, which is usually translated as "control de versiones" instead of "control de fuentes" in the context of source code. However, even with the literal translation, it should be "control de fuente" instead of "fuente de control"